rules/linux/credential_access_potential_linux_ssh_bruteforce.toml

```diff
@@ -0,0 +1,81 @@
+[metadata]
+creation_date = "2022/09/14"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/09/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies multiple consecutive login failures targeting an user account from the same source address and within a
+short time interval. Adversaries will often brute force login attempts across multiple users with a common or known
+password, in an attempt to gain access to accounts.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-system.auth-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential SSH Brute Force Detected"
+note = """## Triage and analysis
+
+#### Possible investigation steps
+- Investigate the login user names.
+- Investigate the source IP address of the failed ssh login attempts.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Identify the source and the target computer and their roles in the IT environment.
+
+### False positive analysis
+- Authentication misconfiguration or obsolete credentials.
+- Service account password expired.
+- Infrastructure or availability issue.
+
+### Response and remediation
+- Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved hosts to prevent further post-compromise behavior.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are
+identified. Reset passwords for these accounts and other potentially compromised credentials, such as email, business
+systems, and web services.
+- Run a full antimalware scan. This may reveal additional artifacts left in the system, persistence mechanisms, and
+malware components.
+- Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
+- Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the
+mean time to respond (MTTR).
+"""
+risk_score = 47
+rule_id = "1c27fa22-7727-4dd3-81c0-de6da5555feb"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Credential Access"]
+type = "eql"
+# query is sequence to 3 runs as the max authentication failure of the linux systems kick in by default after 3 runs.
+# if maxauth tries are overridden the sequence runs can be adjusted accordingly
```

Comment:
I think this type of comment is invalid. You can place it on the query or on the description.
```suggestion
```

Reply:
This was a code comment, to help during query review.